governance.qmd

```diff
@@ -0,0 +1,96 @@
+
+Proposals will be deemed to have been adopted if both one calendar month has passed after notice has been publicly posted and either (i) there has been no further response from an administrator or (ii) dissent by an administrator was recorded and the Proposal has been passed after voting has taken place as per the Voting Process described below.
+
+Proposals to add or remove members of the RMC shall be put forward publicly by any member of the RMC, and shall be treated in the same way as any other Proposal.
+
+### Voting Process
+
+Decision-making is by consensus of the RMC as far as possible. All policy changes, including changes to this document, shall be posted publicly to the `r-multiverse/help` repository as a Proposal. If there is no dissent by another administrator, the Proposal comes into effect one calendar month after it is posted.
+
+If there is dissent by an administrator, they should post a reasoned reply within the one calendar month notice period. However, no one administrator has a veto. Where there is dissent, voting shall take place among administrators to decide on an issue, with a simple majority of votes required for a Proposal to pass. If a Proposal is passed, it will come into effect one calendar month after the Proposal was initially posted. If a Proposal is not passed within one calendar month of the posting of the Proposal, then the Proposal lapses and is not enacted until such a time that it passes.
+
+Voting shall take place publicly under the same Proposal that was initially posted.
+
+As illustration, if the total number of administrators is six, then at least four votes are required to pass a Proposal.
+
+## R-Multiverse Mission
+
+The mission of R-Multiverse is to provide a production-grade, open repository for the R community in general. R-Multiverse is committed to free and equal access, and its source code is released under the open-source MIT License.
+
+### R-Multiverse is non-commercial but commercially-aware
+
+We invite third parties, including for-profit corporations, to contribute to R-Multiverse and use it for their own purposes. However it is important for the longevity and community health of R-Multiverse that the community be recognised as the primary contributor to the project and that the project continues to serve all members of the community equally.
+
+The RMC actively track the corporate allegiances of administrators and moderators, and strives to ensure influence around any particular aspect of the project is not overly skewed towards a single corporate entity.
```

Comment:
allegiances? affiliations?

Reply:
yes, affiliations is a better choice of word :)